lib/manager/composer/artifacts.ts

```diff
@@ -137,7 +135,7 @@ export async function updateArtifacts({
       {
         file: {
           name: lockFileName,
-          contents: await fs.readFile(localLockFileName, 'utf8'),
+          contents: await readLocalFile(lockFileName, 'utf8'),
```

Comment:
```suggestion
          contents: await readLocalFile(lockFileName),
```

I think we don't need it to be string